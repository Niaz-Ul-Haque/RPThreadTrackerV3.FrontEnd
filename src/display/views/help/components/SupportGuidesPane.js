import React from 'react';
import {
	TabPane, CardHeader, CardBlock
} from 'reactstrap';
import Card from '../../../shared/styled/Card';

const SupportGuidesPane = () => (
	<TabPane tabId="guides">
		<Card>
			<CardHeader>
				<i
					className="fas fa-play-circle"
				/> Support Guides
			</CardHeader>
			<CardBlock className="card-body">
				<p>
					Intro Tutorial: How to Use RPThreadTracker ~{' '}
					<a href="https://youtu.be/R7psA53V-9E">Video <i className="fas fa-external-link-alt" /></a>
				</p>
			</CardBlock>
		</Card>
	</TabPane>
);

export default SupportGuidesPane;
