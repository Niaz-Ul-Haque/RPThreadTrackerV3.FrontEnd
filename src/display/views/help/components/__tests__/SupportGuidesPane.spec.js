// #region imports
import React from 'react';
import { shallow } from 'enzyme';
import SupportGuidesPane from '../SupportGuidesPane';
// #endregion imports

const createTestProps = propOverrides => ({
	...propOverrides
});

describe('rendering', () => {
	describe('snapshots', () => {
		it('should render valid snapshot', () => {
			const element = shallow(<SupportGuidesPane {...createTestProps()} />);
			expect(element).toMatchSnapshot();
		});
	});
});
