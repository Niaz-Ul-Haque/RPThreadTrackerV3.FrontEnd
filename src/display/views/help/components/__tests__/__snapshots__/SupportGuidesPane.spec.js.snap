// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering snapshots should render valid snapshot 1`] = `
<TabPane
  tabId="guides"
  tag="div"
>
  <Styled(Card)>
    <CardHeader
      tag="div"
    >
      <i
        className="fas fa-play-circle"
      />
       Support Guides
    </CardHeader>
    <CardBlock
      className="card-body"
      tag="div"
    >
      <p>
        Intro Tutorial: How to Use RPThreadTracker ~
         
        <a
          href="https://youtu.be/R7psA53V-9E"
        >
          Video 
          <i
            className="fas fa-external-link-alt"
          />
        </a>
      </p>
    </CardBlock>
  </Styled(Card)>
</TabPane>
`;
